src/services/PlantingsService.ts

```diff
@@ -20,6 +23,7 @@ const listPlantings = async (
       'delivery.plantings.createdTime',
       'delivery.plantings.numPlants(raw)',
       'delivery.plantings.type',
+      'delivery.plantings.totalPlants',
```

Comment:
nice

Reply:
oops, i was wrong,  i think we need total plants on the plantingSubzones list

delivery.plantings.plantingSubzone.totalPlants

Reply:
Oh, I missed this comment. Will update